packages/dx-react-chart-demos/src/demo-sources/zoom-pan/bootstrap4/scatter.jsx

```diff
@@ -0,0 +1,87 @@
+import * as React from 'react';
+import {
+  ArgumentScale,
+  ValueScale,
+  EventTracker,
+  HoverState,
+} from '@devexpress/dx-react-chart';
+import {
+  Chart,
+  ArgumentAxis,
+  ValueAxis,
+  ScatterSeries,
+  Tooltip,
+  ZoomAndPan,
+} from '@devexpress/dx-react-chart-bootstrap4';
+import { format } from 'd3-format';
+
+import { dataGenerator } from '../../../demo-data/generator';
+
+const adjustDomain = ([start, end]) => [Math.floor(start), Math.ceil(end)];
+
+const formatTooltipValue = format('.2f');
+const TooltipContent = ({ targetItem, data, ...restProps }) => {
+  const item = data[targetItem.point];
+  const arg = item[targetItem.series === 'Series 1' ? 'arg1' : 'arg2'];
+  const val = item[targetItem.series === 'Series 1' ? 'val1' : 'val2'];
+  const text = `${formatTooltipValue(arg)} | ${formatTooltipValue(val)}`;
```

Comment:
It seems to me the text of tooltip is hard to understand. I would expect to see something like this: 
`argument: 0.77`
`value: 0.25`

Reply:
Update. Though I used *X* and *Y* for better alignment.